custom/ewsghana/stock_data.py

```diff
@@ -0,0 +1,51 @@
+from corehq.apps.commtrack.models import SupplyPointCase
+from corehq.apps.locations.models import SQLLocation
+from custom.ewsghana.models import EWSGhanaConfig
+from custom.logistics.stock_data import StockDataSynchronization
+from custom.logistics.tasks import sync_stock_transactions_for_facility
+from dimagi.utils.couch.database import iter_docs
+
+
+class EWSStockDataSynchronization(StockDataSynchronization):
+
+    @property
+    def apis(self):
+        return (
+            ('stock_transaction', sync_stock_transactions_for_facility),
+        )
+
+    @property
+    def all_stock_data(self):
```

Comment:
it might be worth memoizing this - it looks like it's only called once but if that changes it would be silly to get the config from the database each time.